requirements.txt

```diff
@@ -1,4 +1,8 @@
 pytest
-pytest-describe
-git+git://github.com/morganstanley/testplan@latest#egg=testplan; python_version > '3.6'
-plotly
\ No newline at end of file
+pytest-describe; python_version >= '3.6'
+pytest-describe==1.0.0; python_version < '3.6'
+https://github.com/morganstanley/testplan/archive/main.zip; python_version > '3.6'
+plotly
```

Comment:
I think you should add behave package to the requirements, right? Right now I don't see tests that actually check that behave tests are discovered and run. Or am I missing something?

Reply:
I add behave to requirements.txt.
Currently there are no tests that check the behave discovery and run, so no you are not missing something. I try to look into this.